Tests/fixtures/tablet-11.yml

```diff
@@ -7809,3 +7809,131 @@
+  os:
+    name: iPadOS
+    version: 17.5.1
+    platform: ""
+  client:
+    type: mobile app
+    name: Facebook
+    version: 471.0.0.20.112
+  device:
+    type: tablet
+    brand: Apple
+    model: iPad Air 7 WiFi
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: 'Mozilla/5.0 (iPad; CPU OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/21F90 [FBAN/FBIOS;FBAV/471.0.0.20.112;FBBV/618472162;FBDV/iPad14,11;FBMD/iPad;FBSN/iPadOS;FBSV/17.5.1;FBSS/2;FBID/tablet;FBLC/ja_JP;FBOP/5;FBRV/0]'
+  os:
+    name: iPadOS
+    version: 17.5.1
+    platform: ""
+  client:
+    type: mobile app
+    name: Facebook
+    version: 471.0.0.20.112
+  device:
+    type: tablet
+    brand: Apple
+    model: iPad Air 7
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: 'Mozilla/5.0 (iPad; CPU OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/21F90 [FBAN/FBIOS;FBAV/471.0.0.20.112;FBBV/618472162;FBDV/iPad16,3;FBMD/iPad;FBSN/iPadOS;FBSV/17.5.1;FBSS/2;FBID/tablet;FBLC/en_US;FBOP/5;FBRV/619023281]'
+  os:
+    name: iPadOS
+    version: 17.5.1
+    platform: ""
+  client:
+    type: mobile app
+    name: Facebook
+    version: 471.0.0.20.112
+  device:
+    type: tablet
+    brand: Apple
+    model: iPad Pro 5 11" WiFi
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: 'Mozilla/5.0 (iPad; CPU OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/21F90 [FBAN/FBIOS;FBAV/468.0.0.39.104;FBBV/609862394;FBDV/iPad16,4;FBMD/iPad;FBSN/iPadOS;FBSV/17.5.1;FBSS/2;FBID/tablet;FBLC/de_DE;FBOP/5;FBRV/611504180]'
+  os:
+    name: iPadOS
+    version: 17.5.1
+    platform: ""
+  client:
+    type: mobile app
+    name: Facebook
+    version: 468.0.0.39.104
+  device:
+    type: tablet
+    brand: Apple
+    model: iPad Pro 5 11"
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: 'Mozilla/5.0 (iPad; CPU OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/21F90 [FBAN/FBIOS;FBAV/468.0.0.39.104;FBBV/609862394;FBDV/iPad16,5;FBMD/iPad;FBSN/iPadOS;FBSV/17.5.1;FBSS/2;FBID/tablet;FBLC/en_GB;FBOP/5;FBRV/611756927];FBNV/1'
+  os:
+    name: iPadOS
+    version: 17.5.1
+    platform: ""
+  client:
+    type: mobile app
+    name: Facebook
+    version: 468.0.0.39.104
+  device:
+    type: tablet
+    brand: Apple
+    model: iPad Pro 7 12.9" WiFi
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: 'Mozilla/5.0 (iPad; CPU OS 17_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/21F90 [FBAN/FBIOS;FBAV/471.0.0.20.112;FBBV/618472162;FBDV/iPad16,6;FBMD/iPad;FBSN/iPadOS;FBSV/17.5.1;FBSS/2;FBID/tablet;FBLC/en_US;FBOP/5;FBRV/618993062]'
+  os:
+    name: iPadOS
+    version: 17.5.1
+    platform: ""
+  client:
+    type: mobile app
+    name: Facebook
+    version: 471.0.0.20.112
+  device:
+    type: tablet
+    brand: Apple
+    model: iPad Pro 7 12.9"
```

Comment:
iPad Pro 7 13"
it may be worth rounding up to 13, since I did not find the size 12.9 on the apple website (Perhaps this is a purely marketing ploy)

https://www.apple.com/ipad-pro/specs/
https://www.apple.com/ipad-air/specs/

Reply:
The 6th gen has it in the name

https://support.apple.com/en-us/111841

Reply:
Also, see the select from https://www.apple.com/ipad/compare/?modelList=ipad-7th-gen,ipad-8th-gen,ipad-9th-gen

iPad Pro 13" (M4)